crates/bevy_reflect/src/enums/enum_trait.rs

```diff
@@ -126,15 +126,14 @@ pub trait Enum: Reflect {
     }
     /// Returns the full path to the current variant.
     fn variant_path(&self) -> String {
-        format!("{}::{}", self.type_name(), self.variant_name())
+        format!("{}::{}", self.reflect_type_path(), self.variant_name())
```

Comment:
Lol just occurred to me that this wouldn't return a valid path. Not really an issue (and certainly not one this PR should bother fixing). Just wanted to point out my own mistake haha.